classes/fields/currency.php

```diff
@@ -285,6 +285,11 @@ public function validate( $value, $name = null, $options = null, $fields = null,
 	 */
 	public function pre_save( $value, $id = null, $name = null, $options = null, $fields = null, $pod = null, $params = null ) {
 
+		if ( $this->is_empty( $value ) ) {
```

Comment:
`is_empty()` will catch `0` and `0.00` values too, not sure we want that applied here.